cafebabel/templates/articles/edit.html

```diff
@@ -18,17 +18,6 @@ <h1 class="edit {% if article.id %}active{% endif %}">
     <input id=title name=title value="{{ article.title or '' }}" required>
   </h1>
   <div class=article-settings>
-    <div class=select>
```

Comment:
⚠️ articles-tags.js depends on that select, cf. https://github.com/cafebabel/cafebabel.com/pull/305#issuecomment-372165949 to kill one bug with two PRs ;)

Reply:
Not sure what to do with this.
Suggestion @davidbgk?